responsibleai/model_monitor/components/src/calculate_attribution.py

```diff
@@ -0,0 +1,177 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""This file contains the core logic for feature attribution drift component."""
+import numpy as np
+import pandas as pd
+import logging
+
+from responsibleai import RAIInsights
+from sklearn.metrics import ndcg_score
+from ml_wrappers.model.predictions_wrapper import (
+    PredictionsModelWrapperClassification,
+    PredictionsModelWrapperRegression)
+
+try:
+    from lightgbm import LGBMClassifier, LGBMRegressor
+except ImportError:
```

Comment:
What'll happen in this case? In don't see any special handling later on.

Reply:
maybe you can just set a boolean var and throw in the code below if it's invoked to say the env is missing this dependency

Reply:
I pulled this code from https://github.com/microsoft/ml-wrappers/blame/cb7fb17a2ca58d6a8df9d64a6e8d3bdcde2a59f1/tests/common_utils.py and based on git blame it looks like Ilya added this; do you remember why you have try except here (the files from your pr won't load because it's so old : https://github.com/microsoft/ml-wrappers/pull/34/files)